config/config-example.js

```diff
@@ -210,6 +210,15 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// github - this is used for update alerts for a GitHub repository.
+// To use this, uncomment this and fill in the object with the
+// appropriate attributes.
+/**exports.github = {
+ 	port: 3420, // the port for your webhook
+ 	secret: '', // the secret specified during webhook creation
```

Comment:
Whatever default value is here should be rejected at runtime. Some people will just leave default one.